libsql/src/sync.rs

```diff
@@ -537,19 +538,25 @@ async fn try_push(
 
     let mut frame_no = start_frame_no;
     while frame_no <= end_frame_no {
-        let frame = conn.wal_get_frame(frame_no, page_size)?;
+        let batch_size = 128.min(end_frame_no - frame_no + 1);
```

Comment:
We need to make this configurable so that clients with really crappy networking can make it much smaller.

Reply:
Done.